src/utils/v2/__tests__/fundingCycle.test.ts

```diff
@@ -52,7 +52,7 @@ export function packFundingCycleMetadata(
   if (pausePay) packed = packed.or(one.shl(56))
   if (pauseDistributions) packed = packed.or(one.shl(57))
   if (pauseRedeem) packed = packed.or(one.shl(58))
-  if (pauseMint) packed = packed.or(one.shl(59))
+  if (allowMinting) packed = packed.or(one.shl(59))
```

Comment:
@johnnyd-eth this an `pauseBurn` should be swapped

Reply:
@aeolianeth good catch. Done